src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String fileInput = "src/main/resources/dataInputTest.csv";
+    private static FileReaderService readData;
+
+    @BeforeClass
+    public static void setUp() {
+        readData = new FileReaderServiceImpl();
+    }
+
+    @Test
+    public void readFromFile_generaltest_Ok() {
+        File file = new File(fileInput);
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(file));) {
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Unable to open file " + fileInput, e);
```

Comment:
```suggestion
            throw new RuntimeException("Cant read file " + fileInput, e);
```

we could get IOException for many reasons